backend/app/api/routes/documents.py

```
@@ -73,18 +73,21 @@ def remove_doc(
     )
 
     a_crud = OpenAIAssistantCrud(client)
-    d_crud = DocumentCrud(session, current_user.id)
+    d_crud = DocumentCrud(session, current_user.project_id)
     c_crud = CollectionCrud(session, current_user.id)
 
     document = d_crud.delete(doc_id)
     data = c_crud.delete(document, a_crud)
```

Ruff (0.12.2)

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)